README.rst

```diff
@@ -180,7 +180,7 @@ cryptographic keys, and signatures. Whereas the minimal install can only
 verify ed25519 signatures and is intended for sofware updater clients,
 ``tuf[tools]`` provides repository maintainers secure ed25519 key and
 signature verification with `PyNaCl <https://pynacl.readthedocs.io/en/latest/>`_,
-a Python binding to the Networking and Cryptography Library.
+a Python binding to the Networking and Cryptography (NaCl) library.
```

Comment:
I think it's okay as is. That's the format they used on their website:

`PyNaCl: Python binding to the Networking and Cryptography (NaCl) library`

What do you think?